CONTRIBUTING.md

```diff
@@ -71,15 +71,36 @@ git status
 git commit # type a good commit message
 ```
 
-6. Push your commit to GitHub
+6. Bring master up-to-date and rebase
+
+```sh
+git checkout master
+git pull https://github.com/decred/dcrtime.git
+git checkout my-great-stuff
+git rebase -i master
+```
+
+In the text editor, change the command from `pick` to `squash` for **all but the top** commit. If you want to use a different

```

Comment:
I would just note that it's OK, even preferred, if the PR is in separate commits if the steps in the edits are distinct as this can aid code review.  On the other hand, for commits like "oops" or "fix mistake in last thing" or "rebase fixes", those should not exist, not even with an appended message, so use `fix` instead of `squash` for these such commits.